effekt/jvm/src/test/scala/effekt/core/TestRenamer.scala

```diff
@@ -109,12 +109,4 @@ class TestRenamer(names: Names = Names(Map.empty), prefix: String = "") extends
     suffix = 0
     rewrite(s)
   }
-}
-
-object TestRenamer {
-  def rename(b: Block): Block = Renamer().rewrite(b)
-  def rename(b: BlockLit): (BlockLit, Map[Id, Id]) =
-    val renamer = Renamer()
-    val res = renamer.rewrite(b)
-    (res, renamer.renamed)
-}
+}
```

Comment:
Drive-by: this is wrong (should refer to `TestRenamer`, not to `Renamer`), _and_ it's not used => yeet.